pages/team.html

```diff
@@ -37,7 +39,245 @@
             </div>
           {% endfor %}
         </div>
+        <div class="team-map clearfix">
+          <div id="map"></div>
+        </div>
       </div>
       <a target="_blank" href="https://docs.google.com/forms/d/1BAFjC9ON3Zietr0hxtaYpmAfDkuSjx028bM-0veWOUo/viewform" class="button-large">Interested in joining the team?</a>
     </div>
 {% endblock %}
+{% block footer %}
+<script type="text/javascript">
+  function initMap()
+  {
+    var map;
+
+    console.log("InitMap has been hit!!")
+    //Create a new Map.
+    map = new google.maps.Map(document.getElementById('map'),
+            {
+              center: {lat: 0, lng: 0},
+              zoom: 2,
+              disableDefaultUI: true
+            });
+
+    //Create the map styles (disable almost everything except locality labels & set colours)
+    var styles = [
+      { "elementType": "labels", "stylers": [ { "visibility": "off" } ] },
+      { "featureType": "landscape", "stylers": [ { "color": "#21B9D2" },   { "visibility": "simplified" } ] },
+      { "featureType": "water", "stylers": [ { "visibility": "simplified" }, { "color": "#1C99BD" } ] },
+      { "featureType": "administrative", "stylers": [ { "visibility": "off" } ] } ,
+      { "featureType": "poi", "stylers": [ { "visibility": "off" } ] } ,
+      { "featureType": "road", "stylers": [ { "visibility": "off" } ] },
+      { "featureType": "transit", "stylers": [ { "visibility": "off" } ] },
+      { "featureType": "administrative.locality", "elementType": "labels", "stylers": [ { "visibility": "on" } ] }];
+    map.setOptions({styles: styles});
+
+    //Define the pin locations. These were exported from the Reach of rLoop map. Directly using that map seems like a bad idea because it is editable by anyone.
```

Comment:
Actually, I think we should use that map, because we don't want to have to be constantly updating this